src/payment/data/handleRequestError.js

```diff
@@ -95,6 +95,19 @@ export default function handleRequestError(error) {
     ]);
   }
 
+  // Country not DPM compatible
+  if (error.type === 'invalid_request_error' && (
```

Comment:
this error is more specific than the basket changed error. should this come before the basket changed error just in case the previous condition is met somehow unexpectedly?

Reply:
Good idea, will move